pages/atlas/[id].tsx

```diff
@@ -53,6 +54,30 @@ const PostContent: React.FunctionComponent<{
                 </Breadcrumb>
             </Row>
 
+            <Row style={{marginBottom:20}}>
+                {router.query.fromApp === "true" && (
+                    <Button
+                        variant={"link"}
+                        size={"sm"}
+                        style={{fontSize:12}}
+                        onClick={()=> router.back()}
+                    >
+                        {"< Back"}
+                    </Button>
+                )}
+                <Button
+                    variant={"link"}
+                    size={"sm"}
+                    style={{fontSize:12}}
+                    onClick={()=>router.push(getExplorePageURL([{
```

Comment:
@alisman @inodb @onursumer thoughts on this way of creating the filter option on the fly? feels kind of hacky to me but I can't think of a better way to abstract it out so we reuse the logic here and on the explore page, because the explore page logic is buried within multiple components